### ISSUE TITLE
I have the paper key and access to registration email, but forgot passphrase. Can I still recover the files in the filesystem?

### ISSUE THREAD

**Issue by mistnim** [keybase/keybase-issues#3144]
I don't have logged in devices unfortunately.

**Comment by strib** [comment]
You can provision a new device with your paper key, as long as it's not revoked.  Then that new device should be able to access the filesystem.

**Comment by mistnim** [comment]
How do I provision the new device if I can't login?

**Comment by strib** [comment]
You don't need a passphrase to log in, just an existing device, and your paper key counts as a "device".  Just download and install the software on your computer/phone, enter your username and choose to provision with a paper key.

**Comment by mistnim** [comment]
If I run the keybase gui it asks for my email and then for the passphrase I forgot, nothing else I can do there.
If I run keybase device add it says

(Please note that you should run `device add` on a computer that is
already registered with Keybase)
▶ ERROR Login required

**Comment by maxtaco** [comment]
Try your username instead?

**Comment by mistnim** [comment]
@maxtaco yes I managed. Thanks. A bit confusing though, why there are different option given the username or the email?

**Comment by maxtaco** [comment]
CC @patrickxb, there is a privacy concern at this point of the login process that prevents us from revealing which email addresses map to which usernames.  It's artificial though, we can engineer around it with enough elbow grease, but we haven't since it's a pretty uncommon scenario. Unfortunately you hit it!
